Game/GameSinglePlayerScene.swift

```diff
@@ -38,10 +38,24 @@ class GameSinglePlayerScene: SKScene {
 	// MARK: - Events
 	
 	override func keyDown(with event: NSEvent) {
-//		switch event.keyCode {
-//		default:
+		//hardcoded keys for now...
+		switch event.keyCode {
+		case Keycode.leftArrow:
+			game.localPlayerInputMap[.moveLeft].activated = true
+		case Keycode.rightArrow:
+			game.localPlayerInputMap[.moveRight].activated = true
+		case Keycode.downArrow:
+			game.localPlayerInputMap[.moveDown].activated = true
+		case 0x00://US Keyboard 'a'
```

Comment:
Carbon? lolz.

I explicitly didn't include those in Keycode.xyz because they're layout-specific and in a game setting we don't want layout-specific keys. So the hard coded keycodes are temporary anyway until some system is setup for various keyboard layouts. And then that'd be the appropriate place to use something correct — which the Carbon constants don't offer anyway.